packages/file-search/src/node/file-search-service-impl.spec.ts

```diff
@@ -221,7 +221,12 @@ describe('search-service', function (): void {
 
             expect(matchesA).to.not.be.empty;
             expect(matchesB).to.not.be.empty;
-            expect(matchesA).to.deep.eq(matchesB);
+            expect(matchesA.length).to.equal(matchesB.length);
+
+            // Due to ripgrep parallelism we cannot deepEqual the matches since order is not guaranteed.
+            matchesA.forEach(match => {
+                expect(matchesB.includes(match)).to.be.true;
+            });
```

Comment:
What about https://www.chaijs.com/api/bdd/#method_members ?

Reply:
> What about https://www.chaijs.com/api/bdd/#method_members ?

Nice suggestion, I update the pull-request after I confirmed it works :+1: